docs/blog/es-tooling.md

```diff
@@ -0,0 +1,71 @@
+---
+title: ES Tooling - community lead e18e-focused tooling
+author:
+  - name: The e18e Contributors
+sidebar: false
+date: 2024-07-22
+head:
+  - - meta
+    - property: og:type
+      content: website
+  - - meta
+    - property: og:title
+      content: es tooling - community lead e18e-focused tooling
+  - - meta
+    - property: og:url
+      content: https://e18e.dev/blog/es-tooling
+  - - meta
+    - property: og:description
+      content: ES Tooling - a community lead, e18e-focused collection of tools for the ecosystem
+---
+
+# ES Tooling
+
+_July 22, 2024_
+
+The e18e initiative has seen huge growth since launch. We've seen hundreds of performance contributions upstream to popular projects, and many community lead tools.
```

Comment:
```suggestion
The e18e initiative has seen huge growth since launch. We've seen hundreds of performance contributions upstream to popular projects, and the creation of many community lead tools.
```